independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/ClientRequestContextImpl.java

```diff
@@ -225,12 +228,23 @@ public Annotation[] getEntityAnnotations() {
 
     @Override
     public OutputStream getEntityStream() {
-        throw new NotImplementedYet();
+        if (!hasEntity())
+            return null;
+
+        try (ByteArrayOutputStream baos = new ByteArrayOutputStream()) {
+            ObjectOutputStream oos = new ObjectOutputStream(baos);
```

Comment:
Shouldn't represent the entity "on the wire", instead of the java object representation?
I would expect a stream representation of the JSON structure (in case JSON is requested, could also be XML etc.)?
Am I missing something?

Reply:
@famod I could not find any wired entity to get output stream. Wherefrom can I find wired entity?